docs/sources/installation/SUSE.md

```diff
@@ -76,6 +76,20 @@ If you need to add an HTTP Proxy, set a different directory or partition for the
 Docker runtime files, or make other customizations, read our systemd article to
 learn how to [customize your systemd Docker daemon options](/articles/systemd/).
 
+## Uninstallation
+
+To uninstall the Docker package:
+
+    $ sudo zypper rm docker
+
+> **Note**: The above command will not remove images, containers, volumes, or user
```

Comment:
You shouldn't need a note for this. Basically notes:
![image](https://cloud.githubusercontent.com/assets/1347209/7424045/7708bf1c-ef51-11e4-8477-797728a376c5.png)

Since the topic is uninstalling, this applies directly.  So, I'd just:

https://gist.github.com/moxiegirl/b90a8759ebd115f08026